www/disclaimer.html

```diff
@@ -31,17 +31,17 @@
   <script type='text/javascript' src='//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js'></script>
   <![endif]-->
 
-  <link rel="shortcut icon" type="image/x-icon" href="https://www.gov.au/assets/favicon-460a3a91ba3978310658e72a0eeda1625badece48148189cbb1b3aee0bdcd899.ico" />
-  <meta name="csrf-param" content="authenticity_token" />
-<meta name="csrf-token" content="t/uJkD483dk08yC19OI3rG9s1mejaCotEtrkhUv3reajA9fQthSadVOW8YTHMUhQdX+upSW9hj7ilohA7R+w8A==" />
-  <script src="https://www.gov.au/assets/fonts-5cfe6bc8b908837bdc7fec9293f3218caf522af3f672710c8414a04d99165c0a.js" data-turbolinks-track="reload" async="async"></script>
-  <link rel="stylesheet" media="all" href="https://www.gov.au/assets/application-a3be3cb88462fa9da8f5070dff69d461af44aef9098279b1873e41c0b83fa786.css" data-turbolinks-track="reload" />
-  <script src="https://www.gov.au/assets/application-a94daa5c647cc7d06c11b44b120c58ec28337733a939f1cd8d4fc7b8419d1ce9.js" data-turbolinks-track="reload" async="async"></script>
+  <link rel="shortcut icon" type="image/x-icon" href="favico.ico">
+  <meta name="csrf-param" content="authenticity_token">
+<meta name="csrf-token" content="M6D3jM8XwM9nkRYNA2zukGz6l1noMXvowTkqtsyDYf+tp0MBmM4xC2ZxLlJNdOJ2ERPOEqNKhMNTqaoyJ5u0Nw==">
+  <script src="js/fonts-5cfe6bc8b908837bdc7fec9293f3218caf522af3f672710c8414a04d99165c0a.js" data-turbolinks-track="reload" async="async"></script>
+  <link rel="stylesheet" media="all" href="css/application-a3be3cb88462fa9da8f5070dff69d461af44aef9098279b1873e41c0b83fa786.css" data-turbolinks-track="reload">
+  <script src="js/application-a94daa5c647cc7d06c11b44b120c58ec28337733a939f1cd8d4fc7b8419d1ce9.js" data-turbolinks-track="reload" async="async"></script>
 </head>
 
-<body data-ga-id='UA-81570106-1'>
+<body data-ga-id="UA-81570106-1">
 <noscript>
-<iframe height='0' src='https://www.googletagmanager.com/ns.html?id=GTM-NDZ4WX' style='display:none;visibility:hidden' width='0'></iframe>
+<iframe height="0" src="ns.html" style="display:none;visibility:hidden" width="0"></iframe>
```

Comment:
Need to fix this up so it loads from https://www.googletagmanager.com - otherwise google tag manager and analytics will be broken when js is disabled.